JustBrewIt/Pages/Search.cshtml

```diff
@@ -1,35 +1,34 @@
 ﻿@page
 @model JustBrewIt.Pages.SearchModel
 @{
-    var BreweryAPI = (JustBrewItQuickType.Welcome[])ViewData["BreweryAPI"];
+
+    ViewData["Title"] = "Search Page";
```

Comment:
Good.  Go a bit further, and give it context.  "Brewery Search", for instance, as "Search Page" could be on any website.

Reply:
updated.